### PR TITLE
[Fix #41] Use preferred name for rescued variable name

### LINKED ISSUE #41: Rescued exception variable name
After getting a comment from @AleksSenkou in project PR, decided to clarify how we should handle this rule. From my side, having the self-explained variable name (error) is better than just one character variable name (e)
```ruby
# bad
begin
  # do something
rescue MyException => exception
  # do something
end

# good
begin
  # do something
rescue MyException => e
  # do something
end

# good
begin
  # do something
rescue MyException => _e
  # do something
end
```

### PR DESCRIPTION
**Pull request by lazycoder9**

### CODE REVIEW

**Comment by lazycoder9** [comment]
@roman-dubrovsky seems I have to close PR and recreate it from forked repository

**Comment by roman-dubrovsky** [comment]
@lazycoder9 no needed

**Comment by roman-dubrovsky** [comment]
about forking PRs I copied from other gems (for guys who don't have access to editing. Firstly I created it as a personal gem ).

Just make other stuff (Add code style note and note to change log)

File: default.yml [hunk 1]
```diff
@@ -43,3 +43,6 @@ Metrics/LineLength:
 
 Style/StringLiterals:
   EnforcedStyle: double_quotes
+
+Naming/RescuedExceptionsVariableName:
```
Comment (roman-dubrovsky) [inline default.yml]: also, can you order cops by the name

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
 * Change style to `variable` for `Layout/EndAlignment` cop ([@r.dubrovsky][])
 * Change style to `with_fixed_indentation` with indentation width 2 for `Layout/AlignParameter` cop ([@r.dubrovsky][])
 * Always ingone hash aligning for key word arguments (cop: `Layout/AlignHash`) ([@r.dubrovsky][])
+* Use prefered variable name in rescued exceptions (cop: `Naming/RescuedExceptionsVariableName`) ([@ula][])
```
Comment (roman-dubrovsky) [inline CHANGELOG.md]: it should be added to unreleased version (not to 0.1.0). I'll add these changes to `0.2.0` version

Comment (roman-dubrovsky) [reply CHANGELOG.md]: see https://github.com/datarockets/datarockets-style/pull/45/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR9

File: STYLE_GUIDE.md [hunk 3]
```diff
@@ -221,6 +221,25 @@ h = {a: 1, b: 2}
 Array.new(3) { |i| i + 1 }
 ```
 
+* <a name="style-rescued-variable-name"></a>
+  We prefer to use self-explained variable names in rescued exceptions
```
Comment (roman-dubrovsky) [inline STYLE_GUIDE.md]: the current cop pushes us to use `error` as a variable for all exceptions and we can't use other self-explained variable names. 

So what do you think about changing it a little bit. e.g 

```
Use `error` as a variable name on processing exceptions. 
```

**Comment by lazycoder9** [comment]
@roman-dubrovsky Done

**Pull request merged**